product/roundhouse.tests/cryptography/CryptographicServiceSpecs.cs

```diff
@@ -55,7 +55,7 @@ public void should_be_able_to_pass_text_and_get_back_a_base_64_hash_of_the_text(
         [TestFixture]
         public class when_using_an_unofficial_md5_implementation
         {
-            const int passes = 1000000;
+            const int passes = 2000;
```

Comment:
Nice perf improvement! ;)

Reply:
😂

Reply:
I had to sweat for days micro-optimizing that code. Anything for my peeps.